src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -221,6 +222,9 @@
 
     @copy("$PHPUNIT_VERSION_DIR/phpunit.xsd", 'phpunit.xsd');
     chdir("$PHPUNIT_VERSION_DIR");
+    if ($SYMFONY_PHPUNIT_REQUIRE) {
```

Comment:
should be after `$SYMFONY_PHPUNIT_REMOVE`, don't you think?

Reply:
I wasn't sure about it, but yeah, we can change it